src/App.js

```diff
@@ -60,19 +67,63 @@ class App extends Component {
   }
 
   resetAllGames = (type) => {
-    const games = this.state.games;
-    let filteredGames;
+
+    let filterOptions = {
+      type: {
+        card: false,
+        board: false
+      },
+      players: {
+        two: false,
+        threefour: false,
+        fivesix: false,
+        seven: false
+      },
+      age: {
+        lesseight: false,
+        eightthirteen: false,
+        thirteenplus: false,
+        adult: false
+      },
+      genre: {
+        strategy: false,
+        family: false,
+        party: false,
+        adventure: false
+      }
+    }
+
+    document.querySelectorAll('.adv-search-checkbox').forEach(checkbox => {
+      checkbox.checked = false
+    })
+
     document.querySelector('.searchbar').value = '';
     this.closePopUp();
 
-    if (type) {
-      filteredGames = games.filter(game => {
-        return game.genre_ID.includes(type)
-      })
+    let games = this.state.games
+    let filteredGames;
+    let searchPageCheck;
+
+    if (type === 6) {
+      filterOptions.type.board = true;
+      // document.querySelector('#board').checked = true;
```

Comment:
These are commented out because currently we can't set boxes to checked if the AdvancedSearch component is not being rendered, which is why I'm going to try having it always show.